tests/test_confidence.py

```diff
@@ -0,0 +1,74 @@
+import sys
+
+from vulture import core
+
+dc = core.DEFAULT_CONFIDENCE
+
+
+def check_min_confidence(code, min_confidence, expected_dict={}, **expected):
+    v = core.Vulture(verbose=True, min_confidence=min_confidence)
+    v.scan(code)
+    detected = dict((item.name, item.confidence)
+                    for item in v.get_unused_code())
+    expected = expected_dict or expected
+    assert detected == expected
+
+
+def test_confidence_import():
+    code = """\
+import foo
+"""
+    check_min_confidence(code, 50, foo=90)
+
+
+def test_confidence_unreachable():
+    code = """\
+def foo():
+    return
+    bar()
+
+foo()
+"""
+    check_min_confidence(code, 50, {'return': 100})
+    check_min_confidence(code, 100, {'return': 100})
+
+
+def test_function_arg():
+    code = """\
+def foo(a):
+    b = 3
+
+foo(5)
+"""
+    if sys.version_info < (3, 0):
+        check_min_confidence(code, 50, a=dc, b=dc)
+        check_min_confidence(code, dc, a=dc, b=dc)
+    if sys.version_info > (3, 0):
```

Comment:
This won't test anything for Python 3.0.0. Please just use `else` instead of the second `if`.